### PR TITLE
jormungandr: test on ridesharing_journeys added

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.canaltp.fr/browse/NAVP-778

### CODE REVIEW

**Comment by kinnou02** [comment]
tests failed, you will lso have to rebase since my PR change a lot of thing about the journeys that will be returned

**Review by kinnou02 — APPROVED** [review]

**Review by mbounabi — APPROVED** [review]

File: source/jormungandr/tests/instant_system_new_default_routing_tests.py [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        assert distances.get('walking') == 0
+
+        durations = ridesharing.get('durations')
+        assert durations.get('ridesharing') == 926
+        assert durations.get('total') == 926
+
+        rs_sections = ridesharing.get('sections')
+        assert len(rs_sections) == 1
+        assert rs_sections[0].get('mode') == 'ridesharing'
+        assert rs_sections[0].get('type') == 'crow_fly'
```
Comment (pbougue) [inline source/jormungandr/tests/instant_system_new_default_routing_tests.py:186]: Before that, the tests are not mandatory : already tested elsewhere and will change

File: source/jormungandr/tests/instant_system_new_default_routing_tests.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        assert rsj_info.get('seats').get('total') == 4
+
+        rsj_links = rsj_sections[1].get('links')
+        assert len(rsj_links) == 2
+        assert rsj_links[0].get('rel') == 'ridesharing_ad'
+        assert rsj_links[0].get('type') == 'ridesharing_ad'
+
+        assert rsj_links[1].get('rel') == 'tickets'
+        assert rsj_links[1].get('type') == 'ticket'
+        assert rsj_links[1].get('id') == ticket_id
```
Comment (pbougue) [inline source/jormungandr/tests/instant_system_new_default_routing_tests.py:220]: missing tests:
* on tickets (check that link to the section is OK)
* on the prices (ridesharing_journey AND ticket itself)

File: source/jormungandr/tests/instant_system_new_default_routing_tests.py [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+        assert durations.get('total') == 926
+
+        rs_sections = ridesharing.get('sections')
+        assert len(rs_sections) == 1
+        assert rs_sections[0].get('mode') == 'ridesharing'
+        assert rs_sections[0].get('type') == 'crow_fly'
+
+        rs_journeys = rs_sections[0].get('ridesharing_journeys')
+        assert len(rs_journeys) == 1
+        assert rs_journeys[0].get('distances').get('walking') == 16
```
Comment (pbougue) [inline source/jormungandr/tests/instant_system_new_default_routing_tests.py:190]: not sure it's mandatory to test that (we pretty much don't care if it is tracked or not)

**Review by pbougue — COMMENTED** [review]
remarks mostly @kinnou02 for later test rewriting

**Pull request merged**